google/cloud/spanner_dbapi/connection.py

```diff
@@ -196,6 +203,24 @@ def read_only(self, value):
             )
         self._read_only = value
 
+    @property
+    def batch_mode(self):
```

Comment:
If I understand it correctly, giving these names that do not start with an underscore will make them part of the public API. In that case, we should document them and also add validations to verify that they are only called with valid arguments. (But probably we should make them internal instead)

Reply:
Removed the property